specification/overview.md

```diff
@@ -26,21 +26,58 @@ This specification is designed to be used by three major groups:
 
 The FOCUS working group will develop an open-source specification for billing data. The schema will define data dimensions, metrics, a set of attributes about billing data, and a common lexicon for describing billing data.
 
-## Design Notes
+## Design Principles
 
 The following principles were considered while building the specification.
 
-### Provider-Neutral Approach by Default
+### FOCUS is an iterative, living specification
 
-While the schema, naming, terminology and attributes of many providers were reviewed during development, this specification aims to be provider-neutral. In some cases, the approach may closely resemble one or more provider's implementation, while in other cases, the approach might be new. In all cases, the FOCUS group (community composed of FinOps practitioners, Cloud and SaaS providers and FinOps vendors) will attempt to prioritize alignment with the FinOps [Framework][FODOF] and [Capabilities][FODOFC].
+The goal is not to get to a complete, finished specification in one pass. Releasing incremental iterations of the specification will provide value to practitioners and allow feedback as the specification develops.
```

Comment:
I would love for this section to start with a positive statement, not a negative one.  Something like:

```suggestion
Incremental iterations of the specification released on a regular basis will provide higher value to practitioners and allow feedback as the specification develops. The goal is not to get to a complete, finished specification in one pass. 
```

I know that's less snappy, but there's more warm fuzzies, which I think is important this early on in the doc.